### PR TITLE
chore(deps): bump spring zeebe to 8.2.0-alpha2

### PR DESCRIPTION
**Pull request by chillleader**
## Description

Includes the fixes for:
- Metrics https://github.com/camunda-community-hub/spring-zeebe/issues/296
- Inconsistent behavior for disabled webhook https://github.com/camunda-community-hub/spring-zeebe/issues/370

### CODE REVIEW

**Review by igpetrov — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -73,9 +73,9 @@
       </dependency>
 
       <dependency>
-        <groupId>io.camunda</groupId>
+        <groupId>io.camunda.spring</groupId>
         <artifactId>spring-zeebe-connector-runtime</artifactId>
-        <version>${version.spring-zeebe}</version>
+        <version>8.2.0-alpha2</version>
```
Comment (markfarkas-camunda) [inline pom.xml:78]: Just out of curiosity: why was this change necessary? Why can't we resolve the `version.spring-zeebe` property?

Comment (chillleader) [reply pom.xml]: It's not necessary - my bad 🙂 Thanks for pointing out!
